font-src/glyphs/letter/latin/k.ptl

```diff
@@ -599,6 +601,15 @@ glyph-block Letter-Latin-K : begin
 	select-variant 'KHookTop' 0x198 (follow -- 'K')
 	select-variant 'kHookTop' 0x199
 
+	derive-glyphs 'kCaron' 0x1E9 'k' : lambda [src gr] : glyph-proc
+		local shift : Width + SB - Middle + HalfStroke * HVContrast
+		include [refer-glyph 'caronAbove']
+		include : ApparentTranslate shift ((-XH) + Ascender)
+		include [refer-glyph src]
+		include : MarkSet.b
+		set-base-anchor 'aboveBraceL' ((-Middle) + shift - markExtend / 2) aboveMarkMid
```

Comment:
Need to update the placement of `above` anchor too, since we may stack more marks on it.

Reply:
can you show me an example?

Reply:
```
set-base-anchor 'above' ((-Middle) + shift) (Ascender + AccentStackOffset)
```